src/array.jl

```diff
@@ -580,13 +580,6 @@ Return levels which appear in `A`, in the same order as [`levels`](@ref)
 """
 function unique end
 
-"""
```

Comment:
Oh, so the docstring was here originally? Then please keep it here, and not duplicate it below. What I was asking for was actually the current state... Why did you move it?